standards/technical-approach.md

```diff
@@ -16,24 +36,56 @@ TADs do not have to be created for all development work, but MUST be created in
 
-1. The most senior developer on the feature/task/story is assigned a ticket to write the TAD and the ticket MUST be tagged with `tech-approach`
-2. The assigned developer should collect as much information from the BRD, stakeholders and any other relevant information sources and write the TAD
-3. The TAD MAY be circulated at this point for feedback within a team or stakeholders for further input/feedback
+1. An engineer is assigned a ticket to write the TAD and the ticket MUST be tagged with `tech-approach`. Assigning points to a TAD-writing ticket can be challenging. Generally a TAD is given 5 or 8 points, depending on the complexity of the feature and if a [proof of concept](#proof-of-concepts) is necessary.
+2. The assigned developer should collect as much information from the BRD, stakeholders and any other relevant information sources and write the TAD. It is recommended to time box the TAD-writing process to one sprint (2 weeks) so as not to lose context or momentum on the problem at hand. To make this happen, it is recommended to complete a _first_ draft within one week in order to get peer feedback sooner, especially on major challenges and avoid wasting 2 weeks writing about an insufficient approach. Oftentimes the various iterations, including format fixes, corrections, expansions, and summaries, can take longer than expected. If a [proof of concept](#proof-of-concepts) is deemed necessary, then this may extend the TAD-writing phase by an additional week or sprint, depending on the feature or project.
+3. The TAD MAY be circulated at this point for further feedback within a team or stakeholders for further input/feedback. Posting the drafted TAD in a public channel is a good way to start receiving feedback. Scheduling a meeting to review the draft with team members is another option. It is recommended to set clear expectations about how long the TAD will be open for comments and discussion from team members and when that window is considered closed.
 4. When ready for review the Engineering Leadership Team (ELT) should be notified, the TAD uploaded to the shared TAD Google Drive (see below), and the associated JIRA ticket moved to the `Under Review` column
-5. The TAD MUST be reviewed by at least one ELT member with subject area knowledge, and SHOULD be reviewed by two members (see review process below)
-6. If approved in Step 5 the TAD should be sent to Garvita Kapur for final approval. With final approval the associated JIRA ticket is moved to `Done`
-7. If approved specific tickets should be created to execute the work as described in the TAD
+5. The TAD MUST be reviewed by at least one ELT member with subject area knowledge, and SHOULD be reviewed by two members (see review process below) 6. If approved, the TAD MUST be sent to the Director of Engineering for final approval. 7. With final approval the associated JIRA ticket is moved to `Done`. Specific tickets MUST be created to execute the work as described in the TAD.
```

Comment:
Seems like some formatting/line breaks got lost with this update

Reply:
woah you're right good catch